lib/elixir/pages/Guards.md

```diff
@@ -154,13 +157,16 @@ defmodule Check do
   def empty?(_val), do: false
 end
 
-Check.empty?(%{}) #=> true
-Check.empty?({}) #=> false # true was expected!
+Check.empty?(%{})
+#=> true
+
+Check.empty?({})
+#=> false # true was expected!
 ```
 
 This could be corrected by ensuring that no exception is raised, either via type checks like `is_map(val) and map_size(val) == 0`, or by checking equality instead, like `val == %{}`.
 
-It could also be corrected by using multiple guards, so that if an exception causes one guard to fail, the next one is evaluated.
+It could also be corrected by using multiple guards, so that if an exception causes one guard to fail, the next one is still evaluated.
```

Comment:
I am not sure "still" is correct, because the next one is not evaluated if the first one succeeds.

Reply:
I think it is correct, because it is preceded by "if an exception ...", contrasting what happens when no multiple-clauses are used